conf/maestro-conf/edge-config-dell5000-demo.yaml

```diff
@@ -43,19 +43,12 @@ var_defs:
      value: "{{SNAP}}/wigwag/etc/versions.json"
    - key: "UPGRADE_VERSIONS_FILE"
      value: "{{SNAP}}/wigwag/etc/versions.json"
-# devicedb_conn_config:
-#   devicedb_uri: "https://{{ARCH_DEVICE_ID}}:9000" #default uri
-#   devicedb_prefix: "maestro.configs" #default prefix
-#   devicedb_bucket: "lww" #default bucket
-#   relay_id: "{{ARCH_DEVICE_ID}}" #default relay id
-#   ca_chain: "{{SSL_CERTS_PATH}}/ca-chain.cert.pem" #default chain cert file name
-sys_stats: # system stats intervals
```

Comment:
Why are we removing sysstats? Is there a need to remove it?

Reply:
Please refer https://github.com/armPelionEdge/edge-utils/pull/18
Additionally, I think there is no service in the cloud that is listening or gathering these stats (As far as I know)

Reply:
Was informed that sysstats doesn't have a UI and therefore is disabled (also disabled on the raspberry pi). Also too many warnings get generated